packages/story-editor/src/app/taxonomy/taxonomyProvider.js

```diff
@@ -234,7 +225,11 @@ function TaxonomyProvider(props) {
         // We could pull down only the exact term, but
         // we're modeling after Gutenberg.
         if (e.code === 'term_exists') {
-          addSearchResultsToCache(taxonomy, { name: termName }, addToSelection);
+          addSearchResultsToCache(
```

Comment:
cool, this will work nicely with what I had mapped out here: https://github.com/google/web-stories-wp/pull/9212/files#diff-b0d9645393a6258bd68def9cad9560e65e01a48f7631bc87c6a1cb75ce344872 

I do think it's actually `search_items` no `search`

Reply:
`search` is correct

Reply:
great, i'll update it in my pr - it's weird that that was returning me terms when 'search' wasn't. i'll go figure that out this am.